DXMainClient/DXGUI/Multiplayer/ChatListBox.cs

```diff
@@ -27,47 +27,50 @@ private void ChatListBox_DoubleLeftClick(object sender, EventArgs e)
             if (SelectedIndex < 0 || SelectedIndex >= Items.Count)
                 return;
 
-            // Get the clicked link
-            string link = Items[SelectedIndex].Text?.GetLink();
-            if (link == null)
+            // Get the clicked links
+            string[] links = Items[SelectedIndex].Text?.GetLinks();
+            if (links == null)
                 return;
 
-            // Determine if the link is trusted
-            bool isTrusted = false;
-            try
+            foreach (string link in links)
```

Comment:
I don't think double clicking on a message should open multiple links

Reply:
well, FF isn't known for great UI/UX, so we should review features anyways
don't make changes for now, I am not sure how to handle that, accepting ideas

Reply:
Until chat messages would support markdown or highlight link, this implementation is good enough.